nodejs/eks/cluster.ts

```diff
@@ -918,7 +918,13 @@ export function createCore(
                         selectors: selectors,
                         subnetIds: pulumi
                             .output(clusterSubnetIds)
-                            .apply((subnets) => computeWorkerSubnets(parent, fargate.subnetIds ?? subnets)),
+                            .apply((subnets) => {
+                                if (fargate.subnetIds?.length && fargate.subnetIds.length > 0) {
```

Comment:
This change treats undefined and "defined but empty" the same. Is this really correct? Can there be a case where the user might want to express "no subnet ids"?

Reply:
AFAIK that's what we want. If you don't specify (undefined or empty), you fallback to cluster subnets. You cannot give empty list of subnets to prevent falling back to cluster subnets (that's also how AWS API works AFAIK).